tests/test_dot_variants.py

```diff
@@ -16,16 +16,16 @@ def test_systematic_period_square_brackets():
     assert ioc_fanger.fang(s) == "foo.com"
```

Comment:
These tests demonstrate that comma/periods surrounded by only one grouping character (e.g. `{` or `]`) are no longer fanged.